package.json

```diff
@@ -189,6 +192,7 @@
     "puppeteer": "13.7.0",
     "quicktype-core": "6.0.69",
     "regenerator-runtime": "0.13.9",
+    "request": "^2.88.2",
```

Comment:
Should not be needed once https://github.com/angular/angular-cli/pull/23086/files#diff-7c1a5afe62ea7b34798fd2016b3cf81eecfbbf41ebaa2e35f7c884509349372e is merged.